dotnet/src/SemanticKernel.UnitTests/AI/AIExceptionTests.cs

```diff
@@ -40,14 +40,14 @@ public void ItRoundtripsArgsToErrorCodeMessageCtor()
     public void ItRoundtripsArgsToErrorCodeMessageExceptionCtor()
     {
         // Arrange
-        const string Message = "this is a test";
+        const string message = "this is a test";
```

Comment:
Do you have a style guide somewhere that says what style rules are used in the repo?  I'm used to .NET repos where consts are always PascalCased, regardless of whether they're field-level or local-level.

Reply:
I'm currently using the resharper rules in the repo. For constants currently it allows: all caps and camelCase. PascalCase is used in other scenarios. 

We've been talking about automating these checks for people who do not use Rider or Resharper, hopefully that will be in the next sprint. The resharper checks can also be run from the command line. On my side I rely on Rider to identify the issues, e.g. here's a screenshot:

<img width="183" alt="image" src="https://user-images.githubusercontent.com/371009/235324819-bf756825-cdc4-410a-9751-b809123d4441.png">

I think the ideal would be to automate these rules with editorconfig.

Reply:
For reference, here's our style guide for dotnet/runtime:
https://github.com/dotnet/runtime/blob/main/docs/coding-guidelines/coding-style.md

Reply:
> I'm used to .NET repos where consts are always PascalCased, regardless of whether they're field-level or local-level.

we can make the switch if that's more common to .NET, I think we only need a good solution to automate across different platforms. Looks like currently VS/editorconfig is not enforcing any convention for constants.

Reply:
checking if this would work:

```
dotnet_naming_symbols.constant_vars.applicable_kinds = local
dotnet_naming_symbols.constant_vars.applicable_accessibilities = *
dotnet_naming_symbols.constant_vars.required_modifiers = const

dotnet_naming_rule.constant_vars_should_be_pascal_case.severity = error
dotnet_naming_rule.constant_vars_should_be_pascal_case.symbols = constant_vars
dotnet_naming_rule.constant_vars_should_be_pascal_case.style = pascal_case_style
```

Reply:
Whatever style you want to use is fine. It'd just be good if it were documented and automated, as you say :)

Reply:
from a quick look at ASP.NET looks like local const are camelCase, e.g.

https://github.com/dotnet/aspnetcore/blob/main/src/Servers/Kestrel/tools/CodeGenerator/HttpUtilities/HttpUtilities.cs#L247

https://github.com/dotnet/aspnetcore/blob/main/src/Mvc/Mvc.Core/src/Formatters/TextOutputFormatter.cs#L130

Reply:
Yes, different repos have different styles.  We can choose whatever style we want for this repo, it's just desirable to have that style documented / validated via automation.